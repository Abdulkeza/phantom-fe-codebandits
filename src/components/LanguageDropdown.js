import React from 'react';

const LanguageDropdown = ({ styles }) => {
  return (
    <ul className="z-10">
      <li
        className={`${styles} bg-background w-full cursor-pointer hover:font-bold`}
      >
        French
      </li>
      <li
        className={`${styles} top-64 pt-5 lg:pt-3 xl:pt-3 md:top-28 lg:top-28 xl:top-28 bg-background cursor-pointer hover:font-bold`}
      >
        Kinyarwanda
      </li>
    </ul>
  );
};

export default LanguageDropdown;
