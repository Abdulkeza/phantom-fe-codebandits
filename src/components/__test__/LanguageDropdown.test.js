import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import LanguageDropdown from '../LanguageDropdown.js';

const wrapper = shallow(<LanguageDropdown />);

describe('LanguageDropdown', () => {
  it('It is a list of two items', () => {
    expect(wrapper.children().length).toEqual(2);
  });
  it('should render Language Dropdown', () => {
    const elem = renderer.create(<LanguageDropdown />).toJSON();
    expect(elem).toMatchSnapshot();
  });
});
