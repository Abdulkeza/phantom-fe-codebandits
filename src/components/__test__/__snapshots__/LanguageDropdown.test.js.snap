// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LanguageDropdown should render Language Dropdown 1`] = `
<ul
  className="z-10"
>
  <li
    className="undefined bg-background w-full cursor-pointer hover:font-bold"
  >
    French
  </li>
  <li
    className="undefined top-64 pt-5 lg:pt-3 xl:pt-3 md:top-28 lg:top-28 xl:top-28 bg-background cursor-pointer hover:font-bold"
  >
    Kinyarwanda
  </li>
</ul>
`;
